podcasts/New Search/Views/History/SearchHistoryCell.swift

```diff
@@ -85,7 +85,8 @@ struct SearchHistoryCell: View {
                         ZStack {
                             Image("close")
                         }
-                        .frame(width: 48, height: 48)
+                        .frame(width: 56, height: 56)
+                        .background(.red)
```

Comment:
I think you left this in for debugging @SergioEstevao

Reply:
Thanks for the heads up removed it here: https://github.com/Automattic/pocket-casts-ios/pull/3304/commits/b92412d4efcd5df3f400b085efec39288963ce05